components/classes/atm.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/atm",
```

Comment:
We discussed to put these domain specific classes into a more meaningful namespace like 
"https://ns.adobe.com/xdm/fsi/classes/atm"

Any other class which could be used across domains (like product) could stay as such under "https://ns.adobe.com/xdm/classes/..."